libraries/php/phpRiotApi.json

```diff
@@ -0,0 +1,26 @@
+{
+  "owner": "fcarrascosa",
+  "repo": "phpRiotApi",
+  "language": "php",
+  "links": [
+    {
+      "name": "The project on gitHub",
+      "url": "https://github.com/fcarrascosa/phpRiotApi",
```

Comment:
Let's go ahead and removing this trailing comma, for consistency and then we should be good to merge in 😄